### ISSUE TITLE
Explorer is missing/hidden

### ISSUE THREAD

**Issue by RueDeRennes** [microsoft/vscode#195060]
<img width="1192" alt="Bildschirmfoto 2023-10-07 um 12 46 00" src="https://github.com/microsoft/vscode/assets/7523247/d4e7a384-3d5b-4c6d-a3d9-2fe0aae129ee">
Type: <b>Bug</b>

Whenever I open any folder within VSCode, the explorer on the side menu vanishes. It appears that the explorer functionality ceases to work altogether. Neither mouse interactions nor keyboard shortcuts to toggle the explorer seem to rectify this issue.

Steps to Reproduce:

Launch VSCode.
Open any folder via File > Open Folder... or by using the keyboard shortcut (Ctrl+K Ctrl+O on Windows and Linux, ⇧⌘O on macOS).
Observe that the explorer on the side menu disappears.

VS Code version: Code 1.83.0 (Universal) (e7e037083ff4455cf320e344325dacb480062c3c, 2023-10-03T16:13:15.449Z)
OS version: Darwin x64 23.0.0
Modes: Restricted

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-1068NG7 CPU @ 2.30GHz (8 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|3, 3, 3|
<img width="1192" alt="Bildschirmfoto 2023-10-07 um 12 46 00" src="https://github.com/microsoft/vscode/assets/7523247/315c055a-97d5-4099-b3d6-f9ff6587e02c">

|Memory (System)|16.00GB (0.35GB free)|
|Process Argv|--crash-reporter-id bf099403-fb62-44ba-b4e8-77aaa3cca34e|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
ng-template|Ang|16.1.8
vscode-icontheme-nomo-dark|be5|1.3.6
javascript-ejs-support|Dig|1.3.3
gitlens|eam|14.3.0
plantuml|jeb|2.17.5
powerplatform-vscode|mic|2.0.11
fluent-icons|mig|0.0.18
azure-pipelines|ms-|1.228.0
vscode-dotnet-runtime|ms-|1.8.0
azure-account|ms-|0.11.6
vs-keybindings|ms-|0.2.1
vscode-yaml|red|1.14.0
vscode-scss-formatter|sib|3.0.0
vscodeintellicode|Vis|1.2.30

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
89544117:30613380
showlangstatbar:30737416
03d35959:30757346
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxpt:30805730
pythonnoceb:30805159
copilotsettingt:30839829
synctok:30821570
dsvsc013:30795093
dsvsc014:30804076
diffeditorv1:30821571
dsvsc015:30845448

```

</details>

**Comment by gjsjohnmurray** [comment]
Try running the `Reset View Locations` command from the Command Palette (F1).

**Comment by AbdelrahimKoura** [comment]
Thanks https://github.com/gjsjohnmurray

**Comment by GLObus303** [comment]
After the last update, my explorer is missing as well

<img width="219" alt="CleanShot 2023-11-09 at 16 13 49" src="https://github.com/microsoft/vscode/assets/12488648/036e5bdb-a742-46f6-9277-3a2c24ce40eb">

running reset view locations does not help. The only thing that fixes it is showing the activity bar which adds the explorer back to the view and therefore I can use shortcuts to get to it. But it hides back again sooner or later

**Comment by lramos15** [comment]
@GLObus303 This should be fixed in 1.84.1 are you on the latest version?

**Comment by GLObus303** [comment]
Nope, I was 1.84.0. or something less than 1.84.1. Updated. Thanks!
